.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@master
+        with:
+          fetch-depth: 0
+      - name: Setup Node.js 12.x
+        uses: actions/setup-node@master
+        with:
+          node-version: 12.x
+      - name: Install Dependencies
+        run: yarn --frozen-lockfile
+      - name: Create PR or Publish to npm
+        id: changesets
+        uses: changesets/action@master
```

Comment:
Can we pin this so that it cannot be changed underneath our feet so to speak?

Reply:
They don't seem to have any tags/releases https://github.com/changesets/action

Reply:
I suppose we could try to pin it to the latest commit? not sure if the syntax would be `changeset/action#bfeb9e0` or `changeset/action@bfeb9e0`